docs/stargazer.ipynb

```diff
@@ -23,6 +23,7 @@
     "    -   [Display p-values or confidence intervals](#display-p-values-or-confidence-intervals)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    (pf.etable([fit1, fit2, fit3, fit4, fit5, fit6])
</pre><p>When running the nb locally on my maching, I get the following error: </p><p> </p><p><code></code><code>python</p><p><strong>NameError</strong>                                 Traceback (most recent call last)</p><p>Cell <strong>In[30], </strong><a href="about:blank" target="_blank"><strong>line 5</strong></a></p><p>      <a href="about:blank" target="_blank">1</a> (pf.etable([fit1, fit2, fit3, fit4, fit5, fit6])</p><p>      <a href="about:blank" target="_blank">2</a>     .tab_options(column_labels_background_color="cornsilk", </p><p>      <a href="about:blank" target="_blank">3</a>                  stub_background_color="whitesmoke",)</p><p>      <a href="about:blank" target="_blank">4</a>     .tab_style(</p><p><strong>----&gt; </strong><a href="about:blank" target="_blank"><strong>5</strong></a><strong>         style=style.fill(color="mistyrose"),</strong></p><p>      <a href="about:blank" target="_blank">6</a>         locations=loc.body(columns="(3)", rows=["X2"]),</p><p>      <a href="about:blank" target="_blank">7</a>     )</p><p>      <a href="about:blank" target="_blank">8</a> )</p><p> </p><p><strong>NameError</strong>: name 'style' is not defined</p><p></code><code></code></p><p>Do you have an idea what might be going wrong? </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>